UG-Samples/Ports/ActionofPorts/PortDirection/PortDirection/Pages/_Host.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@page "/"
+@using Microsoft.AspNetCore.Components.Web
+@namespace PortDirection.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <base href="~/" />
+    <link rel="stylesheet" href="css/bootstrap/bootstrap.min.css" />
+    <link href="css/site.css" rel="stylesheet" />
+    <link href="PortDirection.styles.css" rel="stylesheet" />
+    <link rel="icon" type="image/png" href="favicon.png"/>
+    <component type="typeof(HeadOutlet)" render-mode="ServerPrerendered" />
+    <script src="_content/Syncfusion.Blazor.Core/scripts/syncfusion-blazor.min.js" type="text/javascript"></script>
```

Comment:
Add the style reference.
<link href="_content/Syncfusion.Blazor.Themes/bootstrap5.css" rel="stylesheet" />